projects/keycloak-angular/src/lib/core/interceptors/keycloak-bearer.interceptor.ts

```diff
@@ -62,22 +63,29 @@ export class KeycloakBearerInterceptor implements HttpInterceptor {
     req: HttpRequest<any>,
     next: HttpHandler
   ): Observable<HttpEvent<any>> {
-    const { enableBearerInterceptor, excludedUrls } = this.keycloak;
+    const {enableBearerInterceptor, excludedUrls} = this.keycloak;
     if (!enableBearerInterceptor) {
       return next.handle(req);
     }
 
-    const shallPass: boolean =
-      excludedUrls.findIndex(item => this.isUrlExcluded(req, item)) > -1;
-    if (shallPass) {
+    if (this.shallPass(excludedUrls, req)) {
       return next.handle(req);
     }
 
     return this.keycloak.addTokenToHeader(req.headers).pipe(
       mergeMap(headersWithBearer => {
-        const kcReq = req.clone({ headers: headersWithBearer });
+        const kcReq = req.clone({headers: headersWithBearer});
         return next.handle(kcReq);
       })
     );
   }
+
+  private shallPass(excludedUrls, req: HttpRequest<any>): boolean {
+
+    if (this.keycloak.enableBearerWhiteListing) {
+      return this.keycloak.includedUrls.findIndex(item => this.urlMatches(req, item)) > -1;
```

Comment:
Here should be equal, right?
```js
return this.keycloak.includedUrls.findIndex(item => this.urlMatches(req, item)) === -1;
```
If the route was not found in included urls, them it should not receive the authorization header.

Reply:
Indeed, ouch... :)